### ISSUE TITLE
"Debranding" CONTRIBUTING.md

### ISSUE THREAD

**Issue by JohnVeness** [aurora-website/website_v1#5]
Currently https://github.com/aurora-website/website_v1/blob/source/CONTRIBUTING.md seems to have lots of references to https://github.com/github/docs that should probably be removed!

**Comment by marchingon12** [comment]
Indeed, I actually haven't edited the [CONTRIBUTING.md](aurora-website/website_v1/blob/source/CONTRIBUTING.md) or rather had the time to rewrite it completely. I know, sounds like a lame excuse but I had many things at hand. It's in the works just like many many fixes so look forward to the next commmit, or you could also contribute! :D

**Comment by JohnVeness** [comment]
Haha, yes, well I considered doing a PR, but it seemed a bit presumptuous of me to edit contributing.md as my first contribution!
